components/service/glean/scripts/sdk_generator.gradle

```diff
@@ -44,15 +63,32 @@ subprocess.check_call([
 
 // Configure the Python environments
 envs {
-    bootstrapDirectory = new File(buildDir, 'bootstrap')
-    envsDirectory = new File(buildDir, 'envs')
+    bootstrapDirectory = GLEAN_BOOTSTRAP_DIR
     pipInstallOptions = "--trusted-host pypi.python.org --no-cache-dir"
 
     // Setup a miniconda environment. conda is used because it works
     // non-interactively on Windows, unlike the other options
-    conda "Miniconda3", "Miniconda3-4.5.11", "64", ["glean_parser==${GLEAN_PARSER_VERSION}"]
+    conda "Miniconda3", "Miniconda3-${MINICONDA_VERSION}", "64", ["glean_parser==${GLEAN_PARSER_VERSION}"]
+}
+
+// Setup the python environment before the build starts.
+task createBuildDir {
```

Comment:
Let's rename this to something more specific: are we creating the miniconda dir or the main glean directory under the gradle home dir here?

Reply:
We're creating the `build` directory for the project here.  That's where the Python plugin insists on downloading the installer for Miniconda.  It ultimately installs it *to* where we tell it to, but it's hardcoded to use the project's build dir as its scratch space for the installer.